custom_dc/load_data.sh

```diff
@@ -0,0 +1,319 @@
+#!/bin/bash
+# Script to process stats using simple stats loader.
+# Defaults
+MODE="customdc"
+OUTPUT_DIR=".data/output_$MODE"
+USAGE="Script to load data into custom dc using simple importer.
+Usage: $(basename $0) [Options]
+Options:
+  -c <file>       Json config file for stats importer
+  -e <file>       Load environment variables from file.
+  -i <dir>        Input directory to process
```

Comment:
I'm wondering if it is easier to use all vars from the env file instead of specifying them as arguments here.

Reply:
Yep.

Reply:
BTW, I wonder if we should even have the input and output directories as environment variables in the file; WDYT?

Reply:
Also, don't we need a way for them to input the GCS bucket as well?

Reply:
The input/output directories can be `gs://<bucket>/<folder>`. 

Is the GCS bucket used elsewhere?

Reply:
Ah, I see; we should mention that.

Reply:
moved -e option to the beginning and added to the usage string.